data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -631,12 +642,13 @@ def _add_score_g(self, df: pd.DataFrame) -> pd.DataFrame:
             (df[self.MEDIAN_INCOME_AS_PERCENT_OF_AMI_FIELD_NAME] < 0.8)
         ) | (df[self.POVERTY_LESS_THAN_100_FPL_FIELD_NAME] > 0.20)
 
-        df["NMTC modified (communities)"] = (
+
+        df["Score K (communities)"] = (
             (df[self.MEDIAN_INCOME_AS_PERCENT_OF_AMI_FIELD_NAME] < 0.8)
-            & (df[self.HIGH_SCHOOL_FIELD_NAME] > high_school_cutoff_threshold)
+            & (df[self.HIGH_SCHOOL_FIELD_NAME] > high_school_cutoff_threshold_2)
```

Comment:
total nit but i think it would be clearer if `high_school_cutoff_threshold_2` were called `high_school_cutoff_threshold_k` to indicate that it's the threshold used in score k. otherwise you have to mentally map between 2 and k when looking at the code.